lib/postProcessing/filter.js

```diff
@@ -62,14 +62,50 @@ filter._filterMatches = function(textToMatch, propertyText) {
 filter._filterKeepObject = function(someObject, filters) {
   for (var k in filters) {
```

Comment:
Can we rename `k` to `attributeName` or something similar that makes clear what it is?